src/migration-failure-worker/migrationFailureWorker.ts

```diff
@@ -9,48 +9,48 @@ import {
   Message,
   pushToFailQueue,
   pushToEmailQueue,
-  areAttemptsAvailable,
+  checkAttemptsAvailability,
   isWaiting
 } from '.';
 
-const { log } = console;
+const { log, error } = console;
 
-export const startWorker = async (
+export const migrationFailureWorker = async (
   config: DotenvParseOutput,
   connection: Connection
 ): Promise<void> => {
-  const worker: Worker = await createWorker(config);
+  const worker: Worker = await createWorker(config).catch(e => error(e));
 
-  const callback = async (message: string, ack: () => void) => {
+  const processFailure = async (message: string, acknowledgment: () => void) => {
     try {
       const parsedMessage: Message = JSON.parse(message);
-
-      log(parsedMessage);
-      log();
+      log('\n', parsedMessage);
 
       const { lastAttempt, attempts = 1 } = parsedMessage;
-      const inWaiting = await isWaiting(config, lastAttempt);
+      const inWaitingPeriod = await isWaiting(config, lastAttempt);
 
-      if (!inWaiting) {
+      if (!inWaitingPeriod) {
         parsedMessage.attempts = attempts + 1;
         await migrate(config, connection, worker, parsedMessage);
-      } else {
-        const inAttempt: boolean = areAttemptsAvailable(config, attempts);
-        console.log(attempts, ' : ', inAttempt);
-        if (inAttempt) {
+      }
+
+      if (inWaitingPeriod) {
```

Comment:
no need for this if statement since at this point, we SHOULD be in waitingPeriod